scripts/on_sd_start.sh

```diff
@@ -63,11 +63,25 @@ case "${OS_NAME}" in
     *)          echo "Unknown OS: $OS_NAME! This script runs only on Linux or Mac" && exit
 esac
 
+# This is the "main" GPU, but some systems will have extra GPUs as "Display Controllers"
+# Notably, AMD laptops with a AMD dGPU, but also probably any desktop with a second GPU.
+# For nvidia it already works fine
+GPU_BRAND=$(lspci 2>/dev/null | grep VGA)
```

Comment:
lspci is not included in the standard installation of many Linux distributions.

Reply:
This could be done without it then, by checking `/sys/class/drm/card?/../../vendor` for the AMD or NVIDIA vendor ID, but that would be more complicated bash scripting i'm not equipped  to do.

Reply:
technically, as i understand the Linux sysfs rules, the most correct way to do this manually with sysfs is to walk up the path from the `card?` path, checking the subsystem of each parent directory, and halting when its the expected subsystem type(in this case, `pci`), because even though theres a `device` link, documentation explictly calls out using it as an application bug.

Reply:
oh, yeah. https://www.kernel.org/doc/html/latest/admin-guide/sysfs-rules.html

> You need to walk up the chain until you find the device that matches the expected subsystem. Depending on a specific position of a parent device or exposing relative paths using ../ to access the chain of parents is a bug in the application.

to be correct it needs to walk